src/components/infomodel-structure/index.tsx

```diff
@@ -210,6 +212,26 @@ const InfoModelStructure: FC<Props> = ({
                     concepts={conceptMap}
                     type={ModelElementType.ROLE}
                   />
+                  {!_.isEmpty(unwrapSpecialization(hasProperty)) && (
```

Comment:
Trenger ikke lodash for dette, er jo bare `array.length > 0`

Reply:
`unwrapSpecialization` returnerer et Partial InformationModelElement, synes selv at lodash er å foretrekke fremfor å sjekke object.keys, men åpen for å bli overbevist :)